templates/zookeeper.properties

```diff
@@ -0,0 +1,21 @@
+clientPort=2181
+dataDir=/var/snap/kafka/common/data
+dataLogDir=/var/snap/kafka/common/log
+dynamicConfigFile=/var/snap/kafka/common/zookeeper-dynamic.properties
+tickTime=2000
+initLimit=5
+syncLimit=2
```

Comment:
Let's remove the current proposed config options, and add these 3 to `config.yaml`.

For the remainder, I think a template is overkill. Let's add it to `src/zookeeper_config.py` as:

```python
DEFAULT_CONFIG_OPTIONS = """
clientPort=2181
dataDir=/var/snap/kafka/common/data
dataLogDir=/var/snap/kafka/common/log
dynamicConfigFile=/var/snap/kafka/common/zookeeper-dynamic.properties
maxClientCnxns=60
minSessionTimeout=4000
maxSessionTimeout=40000
autopurge.snapRetainCount=3
autopurge.purgeInterval=0
reconfigEnabled=true
standaloneEnabled=false
4lw.commands.whitelist=*
DigestAuthenticationProvider.digestAlg=SHA3-256
quorum.auth.enableSasl=true
quorum.auth.learnerRequireSasl=true
quorum.auth.serverRequireSasl=true
authProvider.sasl=org.apache.zookeeper.server.auth.SASLAuthenticationProvider
audit.enable=true
"""

class ZooKeeperConfig:
    # ...

    def set_zookeeper_properties(self) -> None:
        path = "/var/snap/kafka/common/"
        with open(path, "w") as f:
            f.write(DEFAULT_CONFIG_OPTIONS)
"""

Reply:
Or if you really want, dw about changing config at all, and just focus on the TLS stuff and I can do the wider update to modifying config.